src/ec/packages/ec-component-site-header/__snapshots__/site-header.test.js.snap

```diff
@@ -96,6 +96,7 @@ exports[`EC - Site Header Default renders correctly 1`] = `
               </div>
               <div
                 class="ecl-language-list__title ecl-col-12 ecl-col-lg-8 ecl-offset-lg-2"
+                id="ecl-language-list__title"
```

Comment:
Updating site-header snapshots.. Should be site-header-core snapshots instead?

Reply:
This is because the language list comes with an id that we didn't have, i added that in the component which is then included both by the site-header and site-header-core, you see this id now everywhere the language list overlay is used. You can check this markup on ECL:
https://ec.europa.eu/component-library/playground/ec/?path=/story/page-structure-languagelist--overlay

Reply:
This is because the language list comes with an id that we didn't have, i added that in the component which is then included both by the site-header and site-header-core, you see this id now everywhere the language list overlay is used. You can check this markup on ECL:
https://ec.europa.eu/component-library/playground/ec/?path=/story/page-structure-languagelist--overlay

Reply:
ok sorry, I see that the snapshots for the site-header-core are indeed there.